src/debug.c

```diff
@@ -403,6 +403,7 @@ void debugCommand(client *c) {
 "SDSLEN <key> -- Show low level SDS string info representing key and value.",
 "SEGFAULT -- Crash the server with sigsegv.",
 "SET-ACTIVE-EXPIRE <0|1> -- Setting it to 0 disables expiring keys in background when they are not accessed (otherwise the Redis behavior). Setting it to 1 reenables back the default.",
+"SET-SKIP-CHECKSUM-VALIDATION <0|1> -- Enables or disables checksum checks for rdb or RESTORE payload.",
```

Comment:
I've noticed sometimes we add testing stuff here, and other times you add them as configs. Is there a reason we don't always add them here?

Reply:
AFAIK the configs that we want some people to actually use in production are put in CONFIG, and the ones we only add for the purpose of the test suite are put in DEBUG.
The two important differences IMHO are:
1. config options should later be supported forever, we can't remove them. (e.g. no one can argue that we need to keep supporting `SET-ACTIVE-EXPIRE` forever)
2. config options also work right on startup (even before redis starts loading an AOF). this is the reason we sometime put test-suite tunables as config rather than DEBUG (e.g. key-load-delay)

other considerations may be:
- this specific config is dangerous, we don't want it to be enabled with config (we assume DEBUG is blocked in production clusters anyway).